guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -24,8 +19,8 @@ ifndef::orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --disable '*'
-# subscription-manager repos --enable={RepoRHEL7ServerSoftwareCollections} \
---enable={RepoRHEL7Server} --enable={RepoRHEL7ServerSatelliteServerProductVersion}
+# subscription-manager repos --enable={RepoRHEL8Server} \
```

Comment:
```suggestion
# subscription-manager repos --enable={RepoRHEL8BaseOS} \
```
```suggestion
# subscription-manager repos --enable={RepoRHEL8Server} \
```

I wish asciidoc would fail on unknown attributes…

Reply:
Just me your muse (:smile: https://github.com/theforeman/foreman-documentation/pull/1386/commits/5c3a270151fbe3c023d0f94765c8a902e3cfb3e1)